lib/glific/clients/kef.ex

```diff
@@ -285,8 +285,8 @@ defmodule Glific.Clients.KEF do
       {:ok, data} ->
         {key, value} =
           data.json
-          |> Enum.find(fn {k, _v} ->
-            k
+          |> Enum.find(fn {_k, v} ->
+            v["userinputcorrectcode"]
             |> Glific.string_clean()
             |> String.equivalent?(school_id)
```

Comment:
```suggestion
            |> String.equivalent?(Glific.string_clean(school_id))
```

Reply:
@mohitgusain let me know if this PR is good to merge

Reply:
@AkhileshNegi you can merge it. thanks